recipes/shopware6.php

```diff
@@ -76,6 +76,10 @@
     run('cd {{release_path}} && ./bin/build.sh');
```

Comment:
Perhaps we could change the `sw:build` around so it checks whether `build-js.sh` exists or whether shopware version is `>= 6.5`, so we don't need a different application template for this variation.

What are your thoughts?

Reply:
I think it's a good idea to check for build-js.sh so we don't need a different application template for this variation. 
but how can i do that check in a deployer task?

Reply:
You can do stuff like 
```php
if (test('-f {{release_path}}/bin/build-js.sh')) { ... }
```

You can read more here: https://deployer.org/docs/6.x/api#test